changelog/8446-woo-core-verticals

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Prepopulate onboarding form based on Woo Core onboarding
```

Comment:
Should we be more specific here by mentioning what we are prepopulating? We aren't prepopulating anything else at the moment.